imagekitio/file.py

```diff
@@ -83,6 +85,8 @@ def upload(
             raise TypeError(ERRORS.MISSING_UPLOAD_FILENAME_PARAMETER.value)
         url = "%s%s" % (URL.UPLOAD_BASE_URL, "/api/v1/files/upload")
         headers = self.request.create_headers()
+        if isinstance(file, io.BufferedReader):
+            file = base64.b64encode(file.read())
```

Comment:
This is not optimal solution as the API accepts buffer/binary as it is and converting it to base64 is unnecessary.